src/vdaf.rs

```diff
@@ -405,9 +406,15 @@ where
 {
     assert_safe();
 
+    if msgs.is_empty() {
+        return Err(PpmError::Ppm(
+            "verify_finish(): expected at least one inbound messages; got none".to_string(),
+        ));
+    }
+
     // Combine the verifier messages.
-    let mut verifier_data = state.verifier_share.data;
-    let mut joint_rand_seed_check = vec![0; Aes128Ctr::STREAM_CIPHER_SEED_SIZE];
+    let mut joint_rand_seed = state.joint_rand_seed;
+    let mut verifier_data = vec![F::zero(); msgs[0].verifier_share.as_slice().len()];
```

Comment:
meganit: it's a little odd that we use `verifier_share.as_slice()` here, but `verfier_share.data` below, in the iterator. Both achieve the same thing, but I feel like we should either make `Verifier`'s `data` field public, or have a public method allowing access to it, but not both.

Reply:
Good call. Replaced `verifier_share.data` below with `verifier_share.as_slice()` and made `data` private.